src/js/themes/media-theme-demuxed.js

```diff
@@ -0,0 +1,296 @@
+/* 
+<media-theme-demuxed>
+  <video
+    slot="media"
+    src="https://stream.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/high.mp4"
+  ></video>
+</media-theme-demuxed>
+*/
+
+import { defineCustomElement } from '../utils/defineCustomElement.js';
+import MediaTheme from './media-theme.js';
+
+const template = `
+<style>
+  :host {
+    --primary-color: black;
+    --secondary-color: transparent;
+    --tertiary-color: #7596CC;
+  }
+
+  video {
+    object-fit: initial;
```

Comment:
Is this needed? Feels like it shouldn't be.

Reply:
This fixes a difference that shows up between the poster image and the first frame of a video on Safari 15.4+. For the specific case of Demuxed Live this shouldn't be a problem, so I've removed this attribute for now.